articles/support/index.md

```diff
@@ -87,14 +87,14 @@ Ticket response times will vary based on your support plan (shown below).  Note
       <td class="success">Support forum, Support Center, chat, web conference</td>
     </tr>
     <tr>
-      <th>Onboarding Experience</th>
+      <th><a href="https://auth0.com/docs/onboarding/sprint#sprint-benefits-by-support-plan">Onboarding Experience</a></th>
```

Comment:
Can't have absolute links on docs. This needs to be `<a href="/docs/onboarding/sprint#sprint-benefits-by-support-plan">`